test/integration/pack/test_pack.py

```diff
@@ -561,24 +561,25 @@ def test_pack_deb(tt_cmd, tmpdir):
     rc, output = run_command_and_get_output(['docker', 'run', '--rm', '-v',
                                              '{0}:/usr/src/'.format(base_dir),
                                              '-w', '/usr/src',
-                                             'ubuntu',
+                                             'jrei/systemd-ubuntu',
                                              '/bin/bash', '-c',
                                              '/bin/dpkg -i {0} && '
                                              'ls /usr/share/tarantool/bundle1 '
-                                             '&& ls /etc/systemd/system'
```

Comment:
Pls, don't put system units to `/etc`. This is for local resources created by an administrator. Installed units must go to `/usr`. Read `man systemd.unit` for details.

Reply:
Fixed.